doc/setup.md

```diff
@@ -148,16 +172,15 @@ SkyPortal defaults to using port 5000. However, this port may already be in use
 ```
 lsof -i :5000
 ```
-If the command outputs information about a service, it means that port 5000 is already in use. In this case, you may need to configure SkyPortal to use a different port.
+If the command outputs information about a service, it means that port 5000 is already in use. In this case, you may need to configure SkyPortal to use a different port using instructions found in the [Port Configuration](#port-configuration) section.
 
 ## Installation: Debian-based Linux and WSL
 
 1. Install dependencies
 
 	```
 	sudo apt install supervisor postgresql \
-	      libpq-dev python3-pip \
-	      libcurl4-gnutls-dev libgnutls28-dev
+	libcurl4-gnutls-dev libgnutls28-dev
```

Comment:
why the modifications here?

Reply:
I think this may have been a mistake. I'll revert this change.